src/Native/Unix/System.Security.Cryptography.Native/pal_ssl.cpp

```diff
@@ -94,6 +94,10 @@ extern "C" SSL* CryptoNative_SslCreate(SSL_CTX* ctx)
 
 extern "C" int32_t CryptoNative_SslGetError(SSL* ssl, int32_t ret)
 {
+    while(ERR_peek_error() != ERR_peek_last_error())
```

Comment:
I think this needs some code comments explaining what is happening (and why)

Reply:
Good idea, added